web/viewer.css

```diff
@@ -1899,3 +1907,10 @@ html[dir="rtl"] #documentPropertiesOverlay .row > * {
     display: none;
   }
 }
+
+.dir-rtl {
+  direction: rtl;
+}
+.dir-ltr {
+  direction: ltr;
+}
```

Comment:
These rules need to go into the https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js file instead.